native_client/ctcdecode/path_trie.cpp

```diff
@@ -102,20 +99,16 @@ PathTrie* PathTrie::get_path_trie(unsigned int new_char, unsigned int new_timest
   }
 }
 
-void PathTrie::get_path_vec(std::vector<unsigned int>& output, std::vector<unsigned int>& timesteps) {
+void PathTrie::get_path_vec(std::vector<unsigned int>& output) {
   // Recursive call: recurse back until stop condition, then append data in
   // correct order as we walk back down the stack in the lines below.
   if (parent != nullptr) {
-    parent->get_path_vec(output, timesteps);
-  }
-  if (character != ROOT_) {
+    parent->get_path_vec(output);
     output.push_back(character);
-    timesteps.push_back(timestep);
   }
```

Comment:
Here, I realise that I have not completely reverted to the way it was.
It seems to me that it does exactly the same thing with less code, though.
Tell me if you want me to put it back exactly as it was.